components/core/src/clp_s/CMakeLists.txt

```
@@ -97,10 +97,11 @@ if(CLP_BUILD_CLP_S_CLP_DEPENDENCIES)
                 Boost::regex
                 ${CURL_LIBRARIES}
                 fmt::fmt
-                spdlog::spdlog
+		msgpack-cxx
```

```suggestion
                msgpack-cxx
```
Just tab usage like rabbit said, but besides that it looks good.